src/applications/disability-benefits/all-claims/tests/fixtures/data/transformed-data/full-781-781a-8940-test.json

```diff
@@ -326,7 +326,7 @@
             "dates": "A while ago. Weird that this isn't a date field, right?"
           }
         ],
-        "disabilityPreventingEmployment": "Diabetes mellitus1,PTSD (post traumatic stress disorder)",
+        "disabilityPreventingEmployment": "Diabetes mellitus1,PTSD (post-traumatic stress disorder), combat-related",
```

Comment:
Hmm, maybe we need to use a different array joiner? In the `utils/submit.js` file, it's using a plain `.join()`, so changing this to `.join(';')` might be better? The schema looks like it's only expecting a string with no other mentioned requirements or limitations

Reply:
do you think this is causing a failure? Which test(s) do you think would be failing here because of that? I'm not seeing tests failing for submit-transformer anymore, but I see

```
1) Find VA Forms <SearchForm>
       when there is a one-character query added
         should not fetch data and show an error:
     AssertionError: expected null not to be null
      at _callee$ (src/applications/find-forms/tests/containers/SearchForm.unit.spec.jsx:60:7)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)
      at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)
      at asyncGeneratorStep (src/applications/find-forms/tests/containers/SearchForm.unit.spec.jsx:15:103)
      at _next (src/applications/find-forms/tests/containers/SearchForm.unit.spec.jsx:17:194)
      at /home/runner/runner2/_work/vets-website/vets-website/src/applications/find-forms/tests/containers/SearchForm.unit.spec.jsx:17:364
      at new Promise (<anonymous>)
      at Context.<anonymous> (src/applications/find-forms/tests/containers/SearchForm.unit.spec.jsx:17:97)
      at Test.Runnable.run (node_modules/mocha-snapshots/src/index.js:19:22)
      at Hook.Runnable.run (node_modules/mocha-snapshots/src/index.js:19:22)
      at processImmediate (internal/timers.js:461:21)
      at process.callbackTrampoline (internal/async_hooks.js:129:14)

  2) selectVAPServiceInitializationStatus
       returns INITIALIZED if Vet360 is found in the services array:
     Uncaught TypeError: document.createEvent is not a function
      at Object.invokeGuardedCallbackDev (node_modules/react-dom/cjs/react-dom.development.js:3905:[26](https://github.com/department-of-veterans-affairs/vets-website/actions/runs/7226391587/job/19691930837?pr=27156#step:6:27))
      at invokeGuardedCallback (node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at commitRootImpl (node_modules/react-dom/cjs/react-dom.development.js:23091:9)
      at unstable_runWithPriority (node_modules/scheduler/cjs/scheduler.development.js:468:12)
      at runWithPriority$1 (node_modules/react-dom/cjs/react-dom.development.js:11[27](https://github.com/department-of-veterans-affairs/vets-website/actions/runs/7226391587/job/19691930837?pr=27156#step:6:28)6:10)
      at commitRoot (node_modules/react-dom/cjs/react-dom.development.js:2[29](https://github.com/department-of-veterans-affairs/vets-website/actions/runs/7226391587/job/19691930837?pr=27156#step:6:30)90:3)
      at performSyncWorkOnRoot (node_modules/react-dom/cjs/react-dom.development.js:22329:3)
      at /home/runner/runner2/_work/vets-website/vets-website/node_modules/react-dom/cjs/react-dom.development.js:11327:26
      at unstable_runWithPriority (node_modules/scheduler/cjs/scheduler.development.js:468:12)
      at runWithPriority$1 (node_modules/react-dom/cjs/react-dom.development.js:11276:10)
      at flushSyncCallbackQueueImpl (node_modules/react-dom/cjs/react-dom.development.js:11322:9)
      at flushSyncCallbackQueue (node_modules/react-dom/cjs/react-dom.development.js:11[30](https://github.com/department-of-veterans-affairs/vets-website/actions/runs/7226391587/job/19691930837?pr=27156#step:6:31)9:3)
      at scheduleUpdateOnFiber (node_modules/react-dom/cjs/react-dom.development.js:21893:9)
      at Object.enqueueSetState (node_modules/react-dom/cjs/react-dom.development.js:12467:5)
      at Transition.Component.setState (node_modules/react/cjs/react.development.js:365:16)
      at Transition.safeSetState (node_modules/react-transition-group/cjs/Transition.js:[34](https://github.com/department-of-veterans-affairs/vets-website/actions/runs/7226391587/job/19691930837?pr=27156#step:6:35)3:10)
      at /home/runner/runner2/_work/vets-website/vets-website/node_modules/react-transition-group/cjs/Transition.js:290:16
      at Timeout.nextCallback (node_modules/react-transition-group/cjs/Transition.js:[35](https://github.com/department-of-veterans-affairs/vets-website/actions/runs/7226391587/job/19691930837?pr=27156#step:6:36)5:9)
      at Timeout.sentryWrapped [as _onTimeout] (node_modules/@sentry/browser/src/helpers.ts:88:17)
      at listOnTimeout (internal/timers.js:554:17)
      at processTimers (internal/timers.js:[49](https://github.com/department-of-veterans-affairs/vets-website/actions/runs/7226391587/job/19691930837?pr=27156#step:6:50)7:7)
```

Reply:
_re: 1) Find VA Forms <SearchForm>_
I ran `yarn test:unit /Users/verdanceluke/code/vets-website/src/applications/find-forms/tests/containers/SearchForm.unit.spec.jsx` and it passed locally for this branch

Reply:
_re: 2) selectVAPServiceInitializationStatus_

I ran `yarn test:unit /Users/verdanceluke/code/vets-website/src/platform/user/profile/vap-svc/tests/selectors.unit.spec.js` and that passed locally too `¯\_(ツ)_/¯ `

Reply:
Those are flaky tests that I was seeing yesterday as well... I had to re-run CI 8 times to finally get them both to pass